index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Good job working on the "Portfolio: setup and mobile first" 😃, Kindly make sure you stuck the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Your%20website%20should%20look%20exactly%20like%20the%20design%20(e.g.%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) 👍. 

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![Screenshot 2023-05-02 173506](https://user-images.githubusercontent.com/57408419/235714697-a9595aa4-bbc8-48df-908d-979dfd3af9a3.png)| |![Screenshot 2023-05-02 172556](https://user-images.githubusercontent.com/57408419/235712685-9d658fa2-bb41-4a25-b9db-562e33ef5ac9.png)| |